### ISSUE TITLE
xonfig --version and --help arguments error

### ISSUE THREAD

**Issue by laerus** [xonsh/xonsh#1260]
```
> xonsh --version
('xonsh/0.3.4',)
Traceback (most recent call last):
  File "/usr/local/bin/xonsh", line 3, in <module>
    main()
  File "/usr/local/lib/python3.5/dist-packages/xonsh/__amalgam__.py", line 15893, in main
    args = premain(argv)
  File "/usr/local/lib/python3.5/dist-packages/xonsh/__amalgam__.py", line 15852, in premain
    exit()
TypeError: exit() missing 1 required positional argument: 'args'
```

same happens with --help options, it displays the help message and raises the same error

**Comment by scopatz** [comment]
Potential fix in #1266

**Comment by scopatz** [comment]
Thansk for reporting @laerus!
